Bottom: 1c53e9b22121163ca3109c498759a3a1fea9d7a1
Top:    d8543756985c0d0b0065034b1150a772f2df4f77
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 01:58:02 +0000

Avoid clearing pins / vias from polygons... it is already done!





---

diff --git a/src/draw.c b/src/draw.c
index 545c6e9..dd4ff4d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
